sdk/search/search-documents/test/node/searchClient.spec.ts

```diff
@@ -49,229 +49,206 @@ describe("SearchClient", function() {
     }
   });
 
-  describe("#count", function() {
```

Comment:
there's a lot of noise in this file from removing the describe blocks, but I am assuming the test content is all the same

Reply:
Yes. No changes in the contents of the block.